documentation/getting_started.md

```diff
@@ -243,6 +243,37 @@ WARNING: Default profile is used for sign. This signed package is valid for emul
 Package File Location: nnstreamer-example/Tizen.web/ImageClassificationOffloading/.buildResult/ImageClassificationOffloading.wgt
 ```
 
+We need a hybrid app to get the port value automatically from the offloading service.
```

Comment:
What is the hybrid app?
I think it will be helpful if there is more explanation.

Reply:
Added what is hybrid application.
Thank you